src/Core/Commander/Interfaces/ApiInterface/ApiGlobe.js

```diff
@@ -90,7 +90,13 @@ ApiGlobe.prototype.init = function init() {
 ApiGlobe.prototype.addGeometryLayer = function addGeometryLayer(layer) {
     preprocessLayer(layer, this.scene.scheduler.getProtocolProvider(layer.protocol));
     const map = this.scene.getMap();
-    map.layersConfiguration.addGeometryLayer(layer);
+    if (!this.getLayerById(layer.id)) {
```

Comment:
inverse, to avoid the Boolean operator NOT
```js
if (this.getLayerById(layer.id)) {
  // eslint-disable-next-line no-console
    console.error(`Error : id "${layer.id}" already exist`);  
} else {
    map.layersConfiguration.addGeometryLayer(layer);
    this.viewerDiv.dispatchEvent(eventLayerAdded);
}
```
and change comment to more explicit comment
```js
console.error(`Error : id "${layer.id}" already exist, WARNING you layer isn't added`);  
```